pantheon/src/main/java/tech/pegasys/pantheon/services/PantheonPluginContextImpl.java

```diff
@@ -121,9 +121,9 @@ public void registerPlugins(final Path pluginsDir) {
 
   public void startPlugins() {
     checkState(
-        state == Lifecycle.REGISTERED,
+        state == Lifecycle.REGISTERED || state == Lifecycle.STOPPED,
```

Comment:
I'm a bit suspect about this one.  I'm not sure I would have expected the exact same instance of pantheon or its plugins to be able to be restarted.  I'd have expected the `ThreadedPantheonNodeRunner` to create a new instance to start when its restarted.

Reply:
If we really do need this then we need to update the documentation in the plugin API repo to include this change to the life cycle.

Reply:
let me revisit this one.

Reply:
@ajsutton I've reverted changes in this class and incorporated suggestions in ThreadedPan..Runner class.